packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -93,3 +93,9 @@
 	}
 }
 
+.edit-site-global-styles__color-indicator-with-margin {
+	// Add 4px to compensate for the fact that `ColorIndicator` has a height
+	// of 20px, while all other icons have a height of 24px.
+	margin-top: $grid-unit * 0.25;
+	margin-bottom: $grid-unit * 0.25;
```

Comment:
Just a nit, but I find that a "flexed wrapper div set to 24px height" captures the intent better. Or at least a wrapper div with vertical padding, rather than margins on `ColorIndicator` itself.

Reply:
Good point. I'm going to close this conversation as I've decided to adopt an approach based on your suggestion in this  [comment](https://github.com/WordPress/gutenberg/pull/39982#discussion_r841919636)

Reply:
I actually went for this approach in the end, and added a `ColorIndicatorWrapper` component which is basically a `Flex` component with a fixed height of `24px`